packages/create-fuels/src/cli.ts

```diff
@@ -144,6 +159,36 @@ export const runScaffoldCli = async ({
   let projectPath = program.args[0] ?? (await promptForProjectPath());
   const verboseEnabled = program.opts().verbose ?? false;
 
+  const fuelUpSpinner = ora({
+    text: 'Checking if fuelup is installed..',
+    color: 'green',
+  }).start();
+
+  const isFuelUpInstalled = checkIfFuelUpInstalled();
+
+  if (isFuelUpInstalled) {
+    fuelUpSpinner.succeed('fuelup is already installed.');
+  }
+
+  if (!isFuelUpInstalled) {
+    fuelUpSpinner.fail('fuelup not found.');
+    const shouldInstallFuelUp = await promptForFuelUpInstall();
+    if (shouldInstallFuelUp) {
+      const installFuelUpSpinner = ora({
+        text: 'Installing fuelup..',
+        color: 'green',
+      });
+      installFuelUp();
+      installFuelUpSpinner.succeed('Successfully installed fuelup!');
+    } else {
+      log(
+        chalk.yellow(
+          'Warning: You will need to install fuelup manually. See https://docs.fuel.network/guides/installation/#running-fuelup-init'
```

Comment:
I'm afraid a user may miss this, in the scenario where a user has opted to install fuel up and there was a failure, it should be a handled error.
```suggestion
   log(
        chalk.red(
          'Error:  There was an error trying to install `fuelup`, you will need to install `fuelup manually`. See https://docs.fuel.network/guides/installation/#running-fuelup-init' for more details.
 ```

Reply:
@maschad I think adding an error message is 100% a good idea, but only in the case of an actual error if it takes place during the installation. Otherwise, I think we should only log out a warning since this message is only shown when they _choose_ not to have the CLI install `fuelup` for them.

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2149/commits/d89e4bb6921b3801fd93da5fc1d0ceb40218286f

Reply:
Makes sense.